test/test_array.jl

```diff
@@ -139,7 +139,7 @@ const AG = ArchGDAL;
         AG.readraster("ospy/data4/aster_write.img"; flags = AG.OF_UPDATE) do ds
             @test sprint(print, ds) == """
             GDAL Dataset (Driver: HFA/Erdas Imagine Images (.img))
-            File(s): 
```

Comment:
the tests are whitespace sensitive within the strings, so I suspect this will fail